src/toil/wdl/wdl_synthesis.py

```diff
@@ -706,10 +706,13 @@ def write_function_dockercall(self, job):
         stdout = apiDockerCall(self, 
                                image={docker_image}, 
                                working_dir=tempDir, 
-                               parameters=["/root/{job_task_reference}_script.sh"], 
+                               parameters=[os.path.join(tempDir, "{job_task_reference}_script.sh")], 
                                entrypoint="/bin/bash", 
-                               user={docker_user},
-                               volumes={{tempDir: {{"bind": "/root"}}}})
+                               user={docker_user}, 
+                               stderr=True, 
+                               volumes={{tempDir: {{"bind": tempDir}}}})
+        with open(os.path.join(asldijoiu23r8u34q89fho934t8u34fcurrentworkingdir, '{job_task_reference}.log'), 'w') as f:
```

Comment:
haha, what's the story behind this? was this to avoid an accidental name collision?

Reply:
Yes, if you know of a better way, please advise.  XD